.gitignore

```diff
@@ -1,3 +1,5 @@
+# Development tools
```

Comment:
We need to exclude `devel-tools/` AND `/devel-tools/`?

Reply:
Oops, I didn't notice that we already did this in 87a11e070e64c72bb5f9e465354cffebe57ed081.  Fixed.